iothub/service/src/Common/Exceptions/ErrorCode.cs

```diff
@@ -23,47 +30,75 @@ public enum ErrorCode
         BulkRegistryOperationFailure = 400013,
         CannotRegisterModuleToModule = 400301,
 
-        //Unauthorized - 401
+        /// <summary>
+        /// Unauthorized - 401
+        /// </summary>
         IotHubNotFound = 401001,
+
         IotHubUnauthorizedAccess = 401002,
         IotHubUnauthorized = 401003,
 
-        // Forbidden - 403
+        /// <summary>
+        /// Forbidden - 403
+        /// </summary>
         IotHubSuspended = 403001,
+
         IotHubQuotaExceeded = 403002,
         JobQuotaExceeded = 403003,
         DeviceMaximumQueueDepthExceeded = 403004,
         IotHubMaxCbsTokenExceeded = 403005,
 
-        // NotFound - 404
```

Comment:
I think these comments were intentionally not xml comments. They were a sort of grouping for our eyes, since the split up the http error codes by the 3 digit prefix. I'd be inclined to leave them alone rather than convert them to xml comments on a specific error code like you did here

Reply:
I still think it might be worth having them for just not having to suppress them. also, the enum string might be nice to have, I would remove the actual http error code though:
```
///
// Not Found
///
```

Reply:
unless there is a code that is not representing its HTTP counterpart. Which I couldn't find

Reply:
> Looks good, other than the ErrorCode.cs file. The links I gave should help you give better descriptions for at least some of these error codes

I have added all the explanation available. Not everything was documented so I had to add the suppress in the end.